src/main/java/uk/gov/hmcts/divorce/divorcecase/model/CaseData.java

```diff
@@ -243,13 +240,9 @@
     )
     private List<ListValue<ScannedDocument>> scannedDocuments;
 
-    @CCD(
-        label = "Supplementary evidence handled"
```

Comment:
this is used 
<img width="428" alt="image" src="https://user-images.githubusercontent.com/13840402/152016606-93c0c786-c090-4ef2-8765-7fb4c41040fe.png">